src/toil/batchSystems/mesos/batchSystem.py

```diff
@@ -262,6 +265,15 @@ def shutdown(self):
         if driver_result != mesos_pb2.DRIVER_STOPPED:
             raise RuntimeError("Mesos driver failed with %i", driver_result)
 
+    def setEnv(self,name, value=None):
+        """
+        Sets environment variables into dictionary for each job.
```

Comment:
Only need to provide a docstring if the docstring in the overridden method needs to be amended, e.g. if you are modifying the contract defined in that method signature/docstring.